src/shared/popups/gokb-add-title-popup/gokb-add-title-popup.vue

```diff
@@ -52,7 +52,7 @@
                 v-if="id"
                 :label="$t('component.general.uuid.label')"
                 :value="uuid"
-                path="/package-title"
+                path="/#/package-title"
```

Comment:
This does not work for creating an URL which points to the REST API for example. We do not want a link to this client.

Reply:
Hm. That does not make sense to me since the links, if configured correctly, are resolved to client pages anyway.

This comment also applies to the other cases below.